index.md

```diff
@@ -74,7 +95,8 @@ because your past self doesn't answer email.
         transform it for storage with a lossless, well-documented procedure.
     *   Prefer open non-proprietary formats to closed ones (they'll likely last longer)
         *   See [this guide][uiuc-file-formats]
-    *   And don't duplicate contents of stable, long-lived repositories (i.e., don't clone GenBank)
+    *   Consider revoking your own write permission, so it is harder to damage raw data by accident or to hand edit it in a moment of weakness.
```

Comment:
Would "Consider restricting file permissions to read-only even for yourself," be more clear?